src/api/routes/experiment.js

```diff
@@ -37,14 +37,6 @@ module.exports = {
         .catch(next);
     },
   ],
-  'experiment#updateCellSets': [
-    expressAuthorizationMiddleware,
-    (req, res, next) => {
-      experimentService.updateCellSets(req.params.experimentId, req.body)
```

Comment:
What about `experimentService.updateCellSets` do we still need it defined?

Reply:
Nope